r2dec.js

```diff
@@ -29,6 +29,10 @@ const Json64 = require('./decompile/json64.js');
 const r2pipe = require('r2pipe');
 const util = require('util');
 
+if (!r2pipe.jsonParse) {
```

Comment:
Thats not exactly correct. You must check the version numbe in the r2pipe module, or at least check if jsonParse exists in the object instead of just checking for ! Which nay fail if its null for example